README.md

```diff
@@ -181,7 +181,7 @@ Enhancements and Bugfixes should be developed against the ES5 standard and teste
 ## Running / Development
 
 * `npm run server` (Runs a local webserver, visit http://localhost:8080)
-* `npm run tunnel` (Creates an ngrok tunnel, place this in the "Summon 2.0 External Script", e.g. https://9f9981c8.ngrok.io/src/summon/browzine-summon-adapter.js)
+* `npm install -g ngrok && ngrok http 8080` (Creates an ngrok tunnel, place this in the "Summon 2.0 External Script", e.g. https://9f9981c8.ngrok.io/src/summon/browzine-summon-adapter.js)
```

Comment:
Why did we make ngrok a peer dependency?

Reply:
Really good question @KeeganJ! Ubuntu downloads an ngrok binary that slows down the build, so the motivation was to decrease build time. 

However, realizing we should probably keep the dependency local, move it and the `http-server` to `devDependencies`, then during the CircleCI run, we can do an `npm install --production` instead of only `npm install`. That should keep everything lean, using only what's necessary to build and deploy on CircleCI and maybe even speed up a little without the `http-server` install.

Reply:
Sounds good to me!